_source/_posts/2021-11-08-angular-v13.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: blog_post
+title: "What you need to know about Angular v13"
+author: alisa-duncan
+by: advocate
+communities: [security,javascript]
+description: "Highlights of a few Angular v13 features with code snippets"
+tags: [javascript, typescript, angular]
+tweets:
+- "Excited for @angular v13? Check out some of the new features with code demos! #angular"
+- "Check out some use cases for @angular v13 features. What's your favorite feature? #angular"
+- ""
+image:
+type: awareness
+---
+
+Angular v13 arrived! And with it comes a lot of exciting new features and updates. Angular continues to improve run-time performance, decrease compilation time, promote good software development practices, enhance developer experience, and keep up to date with dependencies such as [TypeScript](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/) and [RxJs](https://rxjs.dev/6-to-7-change-summary). Is anyone else excited about RxJS v7?! 🤩 Let's take a look at a few of the many new exciting features in Angular v13. 
+
+## So long, View Engine!
+
+With v13, View Engine is no longer available and replaced by Ivy. Ivy is the future of Angular applications, and the benefits are tremendous. After initially landing in v12, Ivy paves the way for new feature updates, simplifies existing capabilities, and increases rendering performance.
+
+An example of simplifying existing capabilities is how to create components dynamically. Dynamic components are components loaded at runtime. You might use this when you need to tailor visible components based on responses from a server call. Another example might be if your identity token includes a custom claim, such as department, and you want to show a custom dynamic component specific to departmental information. 
```

Comment:
Should we clarify the context of the token example for someone reading this without much knowledge of security, since this is generic Angularb post

```suggestion
An example of simplifying existing capabilities is how to create components dynamically. Dynamic components are components loaded at runtime. You might use this when you need to tailor visible components based on responses from a server call. Another example might be when you are doing OAuth2 authentication, if your identity token includes a custom claim, such as department, and you want to show a custom dynamic component specific to departmental information. 
```

Reply:
Make sure to use "OAuth 2.0" when referring to OAuth.